cmd/cloud_login.go

```diff
@@ -90,6 +104,9 @@ func (c *cmdCloudLogin) run(cmd *cobra.Command, _ []string) error {
 		newCloudConf.Token = null.StringFromPtr(nil)
 		printToStdout(c.globalState, "  token reset\n")
 	case show.Bool:
+		valueColor := getColor(c.globalState.Flags.NoColor || !c.globalState.Stdout.IsTTY, color.FgCyan)
```

Comment:
I moved these lines here, from the bottom of the file, as I think it only makes sense to print the token when providing the `-s` flag, as also discussed at https://github.com/grafana/k6/pull/3886.

Otherwise, it doesn't, because:
- When resetting, with `-r`, there's no token.
- When providing it with `-t`, it's already there.
- When using the form, we define it as a _sensitive_ field (so it's not displayed while being written).

If we want to show it in the last scenario, I'd just switch the form input type to regular text, otherwise (as it was until know) it is a bit incoherent.